### DISCUSSION TITLE
Seeking suggestions for a tool to generate a python client from an openapi spec that will be used within a fastAPI server.

### DISCUSSION THREAD

**Discussion by br-follow** [fastapi/fastapi#8654] [Questions]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
No relevant code
```

### Description

I have an [openapi.yaml spec](https://github.com/alpacahq/bkdocs/blob/master/assets/openapi.yaml) that I would like to use to generate a python client library. This library will be used to make outgoing calls from my FastAPI server. Ideally, it would play somewhat well with the Pydantic typing system used within FastAPI, but that would be a nice to have.

Wondering if people have any suggestions.

### Operating System

macOS

### FastAPI Version

0.70.0

### Python Version

3.9.7

**Comment by ghandic** [comment]
You should look at @dmontagu 's package https://github.com/dmontagu/fastapi_client
